### PR TITLE
vite-plugin-ssr renamed to Vike

### PR DESCRIPTION
**Pull request by brillout**
The vite-plugin-ssr project has been renamed to Vike.
 - [vike.dev](https://vike.dev)
 - [github.com/vikejs/vike](https://github.com/vikejs/vike)

### CODE REVIEW

File: .github/workflows/ecosystem-ci-from-pr.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ on:
           - sveltekit
           - unocss
           - vite-plugin-pwa
-          - vite-plugin-ssr
+          - vike
```
Comment (dominikg) [inline .github/workflows/ecosystem-ci-from-pr.yml:48]: please use alnum sorting here

Comment (brillout) [reply .github/workflows/ecosystem-ci-from-pr.yml:47]: 👍 Done

File: .github/workflows/ecosystem-ci-from-pr.yml [hunk 2]
```diff
@@ -130,7 +130,7 @@ jobs:
...
```
Comment (dominikg) [inline .github/workflows/ecosystem-ci-from-pr.yml:133]: (:

Comment (brillout) [reply .github/workflows/ecosystem-ci-from-pr.yml:132]: Good catch :). PR is updated.

**Review by dominikg — APPROVED** [review]

**Review by lilpoozie2005 — APPROVED** [review]

**Pull request merged**